src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splittedByDotComa = sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item.length > 0);
+
+  const splittedByDoubleDot = splittedByDotComa
+    .map((item) => item.split(':'))
+    .map((SplittedCssProperty) =>
+      // eslint-disable-next-line prettier/prettier
+      SplittedCssProperty.map((propertyPart) => propertyPart.trim()));
```

Comment:
The indentation for the chained methods is inconsistent. According to the checklist, all chained methods should have the same indentation for better readability.

Reply:
All chained methods **_called on the same level_** should have same indentation